docker/snapshots-creator/Dockerfile

```
@@ -1,5 +1,7 @@
 # syntax=docker/dockerfile:experimental
-FROM ghcr.io/matter-labs/zksync-build-base:latest AS builder
+FROM lukemathwalker/cargo-chef:0.1.77-rust-bookworm AS cargo-chef
+
+FROM ghcr.io/matter-labs/zksync-build-base:latest AS chef
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)